PopMovies/PopMovies/Sources/Views/FavoriteMovies/Views/FavoritesMovieViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FavoritesMoviesViewController.swift
+//  PopMovies
+//
+//  Created by Angela Alves on 20/10/22.
+//
+
+import Foundation
+import UIKit
+
+class FavoritesMoviesViewController: PMViewController {
```

Comment:
Acredito que o correto seria apenas a palavra `movies` no plural
```suggestion
class FavoriteMoviesViewController: PMViewController {
```